src/app/modules/item/pages/item-display/item-display.component.html

```diff
@@ -25,11 +27,17 @@
       </ng-template>
     </ng-container>
 
-    <div class="text-center">
+    <div class="text-center error-buttons">
```

Comment:
I think the class `text-center` has no effect anymore since error-buttons has justify-content:center.

Reply:
Fixed